Tests/Spec/tentacle_scenario_04_upgrade_spec.rb

```diff
@@ -35,6 +35,7 @@
   it { should have_policy('Default Machine Policy') }
 end
 
+=begin
```

Comment:
Dont think this should be commented out. It should use the new "check the folder" approach.
It should probably also check the reg entries _are not_ there, as installing a new version should remove them... _I think_